drivers/gles2/rasterizer_storage_gles2.cpp

```diff
@@ -5943,6 +5943,13 @@ void RasterizerStorageGLES2::initialize() {
 	config.pvrtc_supported = config.extensions.has("GL_IMG_texture_compression_pvrtc") || config.extensions.has("WEBGL_compressed_texture_pvrtc");
 	config.support_npot_repeat_mipmap = config.extensions.has("GL_OES_texture_npot");
 
+	// If the desktop build is using S3TC, and you export / run from the IDE for android, if the device supports
+	// S3TC it will crash trying to load these textures, as they are not exported in the APK. This is a simple way
+	// to prevent Android devices trying to load S3TC, by faking lack of hardware support.
+#if !defined(TOOLS_ENABLED) && defined(ANDROID_ENABLED)
```

Comment:
As discussed, you could likely remove `TOOLS_ENABLED` and add `IPHONE_ENABLED` for good measure, just in case (probably no device supporting it but who knows).